test/graph.test.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+
+import { describe, expect, test, beforeEach } from 'vitest';
+import { Project } from 'fixturify-project';
+import { runCommandFactory, setupProject, resolveCLIBin } from './test-helpers';
+
+describe('validation-test @rehearsal/cli graph', () => {
+  let run;
+
+  beforeEach(async () => {
+    let project = new Project('simple', '1.0.0', {
+      files: {
+        'index.js': `
+          import './lib/hotdog';
+          import './lib/burger';
+        `,
+        lib: {
+          'hotdog.js': `import './sandwich';`,
+          'burger.js': `import './sandwich';`,
+          'sandwich.js': `import 'chalk';`
+        }
+      }
+    });
+    project.pkg.exports = './index.js';
+    project.pkg.volta = {
+      node: '16.19.0'
+    };
+    project.addDependency('chalk', '5.2.0');
+
+    // Setup project with dependencies to use rehearsal e.g. typescript, eslint, prettier
+    await setupProject(project);
+
+    const bin = resolveCLIBin(project);
+
+    // Set up command for tests
+    run = runCommandFactory(bin, { cwd: project.baseDir });
+  });
+
+  test('graph --help', () => {
+    const results = run(['graph', '--help']);
+    // Dumping the output for debuggability
+    console.log(results.stdout);
```

Comment:
delete all the console.log's

Reply:
I left the console.log's in to make it easier to debug the output from a failing smoke-test. If I want something added for debuggability, I should just use debug package and build a callback, then we can change the smoke-test GH action configure that.

I will remove for now, if we need it I will add debug later.